.github/ci.sh

```diff
@@ -133,9 +133,9 @@ build_yices() {
     export CONFIGURE_FLAGS="--prefix=$TOP/install-root"
   fi
 
-  mkdir install-root
-  mkdir install-root/include
-  mkdir install-root/lib
+  mkdir -p install-root
```

Comment:
Now that we're using `mkdir -p`, is this first command necessary? I would expect the latter two commands to create `install-root` if it hasn't been created already.